### PR TITLE
[ZEPPELIN-4886] [flink] Add property to start flink job from savepoint

### PR DESCRIPTION
**Pull request by lonelyGhostisdog**
### What is this PR for?
Add property to start flink job from savepoint

### What type of PR is it?
[Improvement]

### Todos
* [ ] - Task

### What is the Jira issue?
[ZEPPELIN-4886] https://issues.apache.org/jira/browse/ZEPPELIN-4886

### How should this be tested?
GUI integration tests

### Questions:
No Questions.

### CODE REVIEW

**Comment by lonelyGhostisdog** [comment]
> Thanks for the contribution @lonelyGhostisdog have you tested it manually ?
> 2 cases: one is setting the savepointPath correctly, anther case is that setting an invalid path.

Hi,zjffdu
I have test both of them 
when i set the savepointPath correctly , it's ok 
when i set the savepointPath incorrectly , it shutdown and throw execption 
And i commit the test method just now , PTAL

File: flink/interpreter/src/test/java/org/apache/zeppelin/flink/FlinkStreamSqlInterpreterTest.java [hunk 1]
```diff
@@ -253,6 +254,67 @@ public void testResumeStreamSqlFromSavePoint() throws IOException, InterpreterEx
...
+    Thread thread = new Thread(() -> {
+      try {
+        InterpreterContext context = getInterpreterContext();
+        context.getLocalProperties().put("type", "update");
+        context.getLocalProperties().put("savepointDir", savePointDir.getAbsolutePath());
+        context.getLocalProperties().put("parallelism", "1");
+        context.getLocalProperties().put("maxParallelism", "10");
+        InterpreterResult result2 = sqlInterpreter.interpret("select url, count(1) as pv from " +
+                "log group by url", context);
+        System.out.println("------------" + context.out.toString());
```
Comment (zjffdu) [inline flink/interpreter/src/test/java/org/apache/zeppelin/flink/FlinkStreamSqlInterpreterTest.java:275]: Don't use System.out, use log4j instead

File: flink/interpreter/src/test/java/org/apache/zeppelin/flink/FlinkStreamSqlInterpreterTest.java [hunk 2]
```diff
@@ -253,6 +254,67 @@ public void testResumeStreamSqlFromSavePoint() throws IOException, InterpreterEx
             resultMessages.get(0).getData().contains("url\tpv\n"));
   }
 
+  @Test
+  public void testResumeStreamSqlFromExistSavePointPath() throws IOException, InterpreterException, InterruptedException, TimeoutException {
```
Comment (zjffdu) [inline flink/interpreter/src/test/java/org/apache/zeppelin/flink/FlinkStreamSqlInterpreterTest.java:258]: Could you add another test that resume from invalid savepointPath ?

Comment (lonelyGhostisdog) [reply flink/interpreter/src/test/java/org/apache/zeppelin/flink/FlinkStreamSqlInterpreterTest.java:258]: I will finish them soon

**Comment by zjffdu** [comment]
@lonelyGhostisdog Could you paste the travis CI link ?

**Comment by lonelyGhostisdog** [comment]
> @lonelyGhostisdog Could you paste the travis CI link ?

Hi,zjffdu
https://travis-ci.org/github/lonelyGhostisdog/zeppelin

**Pull request closed without merging**